### ISSUE TITLE
Enter Commit's Files of Repo First Commit will Popup Error Dialog

### ISSUE THREAD

**Issue by azzgo** [jesseduffield/lazygit#1976]
**Describe the bug**
A Error of Git command output tips  "unknown revision or path not in the working tree".

**To Reproduce**
1. Go to '4 Commits - Reflogs Section '
3. Scroll down to 'first commit of the git commits'
4. Click on '<Enter>' to Commits files section
5. See error window

the dialog Info

```
fatal: ambiguous argument 'xxxxx^': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

**Expected behavior**
The Repo First commit files list and related added lines highlight in Patch Section

**Screenshots**

<img width="1827" alt="image" src="https://user-images.githubusercontent.com/5636512/171578410-9de26526-4551-4e60-a2b3-2882e825f1ec.png">

<img width="1825" alt="image" src="https://user-images.githubusercontent.com/5636512/171577823-5327d792-a058-4b39-916d-5db8f5219043.png">

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS 12.4]
 - Lazygit Version [e.g. v0.34]

**Additional context**
I also trigger in other git repo, and my arch linux mathine.

**Comment by mark2185** [comment]
This was fixed in #1944  but that was _after_ the last release, could you give the current `master` a spin and see if that helps?

**Comment by azzgo** [comment]
Oh, cool, it works.

Because I searched, not found similar issue.😂

Thank you for reply.
